rust_build_utils/rust_utils_config.py

```diff
@@ -66,11 +66,11 @@
     "windows": {
         "archs": {
             "x86_64": {
-                "rust_target": "x86_64-pc-windows-gnu",
+                "rust_target": "x86_64-pc-windows-msvc",
```

Comment:
This is a breaking change for anyone cross compiling on Linux. At the moment I see that Libdrop is cross compiled from Linux. This is not a blocker, but should be mentioned in CHANGELOG.md, and major version should be bumped.

Reply:
There's no changelog file here, and we agreed with tech leads to do this breaking change. ARM64 support will require MSVC toolchain and a windows runner anyways

Reply:
So create `CHANGELOG.md` and mention that starting with version `X.X.X` cross compiling Windows is not supported anymore and compiling for Windows has to be done on Windows host.